model/registry/gen-ai.yaml

```diff
@@ -45,6 +45,26 @@ groups:
         brief: The top_p sampling setting for the LLM request.
         examples: [1.0]
         tag: llm-generic-request
+      - id: request.top_k
+        stability: experimental
+        type: double
+        brief: The top_k sampling setting for the LLM request.
+        examples: [1.0]
+      - id: request.stop_sequences
+        stability: experimental
+        type: string[]
+        brief: The stop sequences provided in the request.
+        examples: ['\n']
+      - id: request.frequency_penalty
```

Comment:
Would this be a candidate for future attributes under `frequency`? Also the one below with `presence`? Should we reserve these namespaces already?